packages/core/src/database/index.ts

```diff
@@ -106,30 +106,34 @@ type PGliteDriver = {
 type PostgresDriver = {
   internal: Pool;
   user: Pool;
-  readonly: Pool;
   sync: Pool;
+  readonly: Pool;
 };
 
 type QueryBuilder = {
   /** For updating metadata and handling reorgs */
   internal: HeadlessKysely<PonderInternalSchema>;
   /** For indexing-store methods in user code */
   user: HeadlessKysely<any>;
-  /** Used in api functions */
-  readonly: HeadlessKysely<unknown>;
   /** Used to interact with the sync-store */
   sync: HeadlessKysely<PonderSyncSchema>;
+  /** Used in api functions */
+  readonly: HeadlessKysely<unknown>;
+  /** Used in client queries */
+  // client: HeadlessKysely<unknown>;
+  drizzle: Drizzle<Schema>;
+  drizzleReadonly: Drizzle<Schema>;
 };
 
-export const createDatabase = ({
+export const createDatabase = async ({
```

Comment:
Small nit - it's getting difficult to remember what happens in `createDatabase` vs `database.setup`. Maybe need a new name for setup - `prepareNamespace` ?